dist/stat_data.py

```diff
@@ -339,8 +339,9 @@ def __init__(self, name, desc, flags=''):
     # Chunk cache statistics
     ##########################################
     ChunkCacheStat('chunk_cache_bytes_inuse', 'total bytes used by the cache'),
-    ChunkCacheStat('chunk_cache_chunks_inuse', 'total chunks held by the chunk cache'),
     ChunkCacheStat('chunk_cache_chunks_evicted', 'chunks evicted'),
+    ChunkCacheStat('chunk_cache_chunks_inuse', 'total chunks held by the chunk cache'),
+    ChunkCacheStat('chunk_cache_chunks_pinned', 'total number of pinned chunks cached in the chunk cache'),
```

Comment:
Do you think we should include a similar stat for bytes? We have `chunk_cache_bytes_inuse` but nothing similar for pinned. Genuine question, not intended as a suggestion.

Reply:
Yes, this stat is certainly very useful. I think we should have a variable in the struct called `pinned_bytes_used`. Perhaps this will be useful if we later wish to evict pinned objects.

Reply:
OK cool. I suspect it's also handy for TS - I can imagine it being annoying to need to figure out why the chunk cache is huge, then having to do the extra work of figuring out the chunk size, then multiplying that by the chunk count.

Reply:
True, updated in [efc862d](https://github.com/wiredtiger/wiredtiger/pull/9480/commits/efc862d902141e082a164839bb362d0ea06e5503)